tensorflow_lite_support/ios/test/task/vision/utils/sources/GMLImage+Helpers.m

```diff
@@ -28,4 +28,35 @@ + (GMLImage *)imageFromBundleWithClass:(Class)classObject
   return [[GMLImage alloc] initWithImage:image];
 }
 
+- (CVPixelBufferRef)grayScalePixelBufferFromUIImage {
```

Comment:
This is better as a category of UIImage rather than GMLImage. Could you move it to GMLImageUtils?

Reply:
For the time being I have removed this from the PR since it is needed for tests. I'll updated it in the follow up PR when the tests are added.